content/extraProblems.json

```diff
@@ -589,6 +589,149 @@
         "usacoId": "1166"
       }
     },
+    {
+      "uniqueId": "usaco-1179",
+      "name": "Herdle",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1179",
+      "source": "Bronze",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["Simulation"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1179"
+      }
+    },
+    {
+      "uniqueId": "usaco-1180",
+      "name": "Non-Transitive Dice",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1180",
+      "source": "Bronze",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Brute Force"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1180"
+      }
+    },
+    {
+      "uniqueId": "usaco-1181",
+      "name": "Drought",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1181",
+      "source": "Bronze",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Greedy", "Math"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1181"
+      }
+    },
+    {
+      "uniqueId": "usaco-1183",
+      "name": "Cow Frisbee",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1183",
+      "source": "Silver",
+      "difficulty": "Normal",
+      "isStarred": false,
+      "tags": ["Sorted Set", "Stack", "Linked List"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1183"
+      }
+    },
+    {
+      "uniqueId": "usaco-1184",
+      "name": "Cereal 2",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1184",
+      "source": "Silver",
+      "difficulty": "Very Hard",
+      "isStarred": false,
+      "tags": ["Spanning Tree", "Constructive"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1184"
+      }
+    },
+    {
+      "uniqueId": "usaco-1185",
+      "name": "Drought",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1185",
+      "source": "Gold",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["DP"],
+      "solutionMetadata": {
+        "kind": "USACO",
+        "usacoId": "1185"
+      }
+    },
+    {
+      "uniqueId": "usaco-1186",
+      "name": "Farm Updates",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=1186",
+      "source": "Gold",
+      "difficulty": "Hard",
+      "isStarred": false,
+      "tags": ["Connected Components"],
```

Comment:
dsu maybe

Reply:
The official solution doesn't use it.

Reply:
so? that doesn't mean you can't solve it w/ dsu?

Reply:
I don't see a reason to tag it with DSU if DSU isn't necessary to solve it and doesn't make the solution any simpler.

Reply:
so what you're saying is alternative methods should only be mentioned if they make the solution simpler?

Reply:
Yes.

1. I do not see how a solution involving DSU would be simpler than the solution in the official analysis.
2. If the only criterion for tagging a problem with DSU was if DSU could potentially be used, then (almost?) every problem involving connected components would also be tagged with DSU.
3. DSU is considered a Gold-level topic. By not tagging it with DSU, I hope I made it clear that this problem could very well have been used in Silver (in place of Cereal 2).

Reply:
lmao sure

Reply:
i mean i still disagree but you make some good points
the pr's merged anyways lol